tests/test_two_different.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 D-Wave Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import subprocess
+import unittest
+
+example_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+class TestSmoke(unittest.TestCase):
+    # test that the example runs without failing
+    def test_smoke(self):
```

Comment:
ditto with the naming

Reply:
Fixed.

Reply:
Has the changed been pushed yet? I don't see the change here in the code. It still is called `TestSmoke` and `test_smoke`.

![image](https://user-images.githubusercontent.com/5600582/76339904-f8be9c00-62b7-11ea-8df5-51dac45b79b9.png)